### DISCUSSION TITLE
Wrapper with Components

### DISCUSSION THREAD

**Discussion by aheissenberger** [infinum/eightshift-frontend-libs#705] [Q&A]
How can I wrap the header (layout-three-columns) with the wrapper?

```php
<?php
	// Header Component.
	echo Components::render(
		'layout-three-columns',
		[
			'layoutThreeColumnsHtmlTag' => 'nav',
			'additionalClass' => 'header--sticky',
			'layoutThreeColumnsLeft' => Components::render(
				'logo',
...
```

**Comment by mbmjertan** [comment]
Looks like the team missed your question (new discussions don't notify repo watchers - keep an eye on them @goranalkovic-infinum @iruzevic or disable them).

Unfortunately you can not render the wrapper like you would render a component. It would be very useful, though, and it was discussed a lot previously. I would suggest that you create a layouting component that provides the layout structure your design needs, and then use that. For a lot of flexibility, if you're willing to tolerate the code smell of doing that, there's nothing preventing you from copying the wrapper and making it into a component...
